src/sentry/static/sentry/app/stores/repositoryStore.tsx

```diff
@@ -0,0 +1,88 @@
+import Reflux from 'reflux';
+
+import RepoActions from 'app/actions/repositoryActions';
+import {Repository} from 'app/types';
+
+type RepositoryStoreInterface = {
+  get(
+    orgSlug?: string
+  ): {
+    repos: Array<Repository> | undefined;
+    reposLoading: boolean | undefined;
+    reposError: Error | undefined;
+  };
+
+  orgSlug: string | undefined;
+  lastUpdated: number;
+  repos: Array<Repository> | undefined;
```

Comment:
Why array and not `Repository[]`

Reply:
@evanh we should use https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/array-type.md#array-simple with "array-simple"

Reply:
There are cases where maybe we want to use the `Array<>` syntax, but mostly I think we probably should be using `Thing[]`.

Reply:
Also I'm @EvanPurkhiser :)

Reply:
Hello @EvanPurkhiser, I'm @leedongwei. 

I like the `array-simple` rule. @scttcper you should propose during the TSC that we add it into our ESLint rules.